fitapp/migrations/0005_upgrade_oauth1_tokens_to_oauth2.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations, models
+
+from fitbit.api import FitbitOauth2Client
+from fitapp.utils import get_setting
+from oauthlib.oauth2.rfc6749.errors import MissingTokenError
+
+
+def forwards(apps, schema_editor):
+    UserFitbit = apps.get_model('fitapp', 'UserFitbit')
+    for fbuser in UserFitbit.objects.filter(refresh_token=''):
+        try:
+            token = FitbitOauth2Client(
+                get_setting('FITAPP_CONSUMER_KEY'),
+                get_setting('FITAPP_CONSUMER_SECRET'),
+                refresh_token='{0}:{1}'.format(
+                    fbuser.access_token, fbuser.auth_secret)
+            ).refresh_token()
+            fbuser.access_token = token['access_token']
+            fbuser.refresh_token = token['refresh_token']
+            fbuser.save()
+        except MissingTokenError:
```

Comment:
@brad I think that its better to delete the user because that way the user can re-sync his/her device and get new tokens for future requests. Otherwise, we won't be able to get fitbit data for that user